tests/e2e/default/analytics/analytics.wdio-spec.js

```diff
@@ -61,14 +61,14 @@ describe('Targets', () => {
     const targets = await analyticsPage.getTargets();
 
     expect(targets).to.have.deep.members([
-      { title: 'Deaths', goal: '0', count: '0' },
-      { title: 'New pregnancies', goal: '20', count: '0' },
-      { title: 'Live births', count: '0' },
-      { title: 'Active pregnancies', count: '0' },
-      { title: 'Active pregnancies with 1+ routine facility visits', count: '0' },
+      { title: 'Deaths', goal: '0', count: '0', countNumberColor: '#76b0b0' },
```

Comment:
Please extract these colours to a constant.
This constant uses uppercase & snakecase. 
It can be in the `analyticks.wdio.page.js` and exported to access here.

Reply:
Hi @latin-panda does this work ? If it does I can update the rest
<img width="846" alt="Screenshot 2023-10-30 at 11 00 38" src="https://github.com/medic/cht-core/assets/2597305/4c8c5dcd-e213-4a75-ba50-0ead6c434b8a">

Reply:
What about a bit shorter like this:
TARGET_UNMET_COLOR
TARGET_MET_COLOR

met = achieved = reached --> I'll leave the proper term on you :)